Tiltfile

```diff
@@ -71,24 +89,5 @@ k8s_yaml(helm(
     values=['./charts/egressd/values-tilt.yaml']
 ))
 
-helm_remote(
-    'grafana',
-    repo_url='https://grafana.github.io/helm-charts',
-    repo_name='grafana',
-    version='6.50.7',
-    namespace=namespace,
-    set=[],
-    values=['./hack/grafana-tilt-values.yaml']
-)
-
-helm_remote(
-    'victoria-metrics-single',
```

Comment:
why it was removed? 🤔

Reply:
Do we use it?

Reply:
I was using it for testing purposes when data is not going to CAST :) also, it will be used by default by the exporter when no http config is present https://github.com/castai/egressd/blob/ab27191134ea83c616acff9bedafaaa6f34534c6/cmd/exporter/main.go#L131